pipeline/api/cloud.py

```diff
@@ -27,9 +27,16 @@ class PipelineCloud:
     token: Optional[str]
     url: Optional[str]
 
-    def __init__(self, url: str = None) -> None:
-        self.token = os.getenv("PIPELINE_API_TOKEN")
+    def __init__(self, url: str = None, token: str = None) -> None:
+        self.token = token or os.getenv("PIPELINE_API_TOKEN")
         self.url = url or os.getenv("PIPELINE_API_URL", "https://api.pipeline.ai")
+        if self.token is not None:
+            self.authenticate()
+        else:
+            print(
+                "No token set, please set one and invoke \
+                     PipelineCloud.authenticate() method"
```

Comment:
This notation will result in spaces in the messages.

```suggestion
                "No token set, please set one and invoke PipelineCloud.authenticate() method"
```